llvm/lib/Transforms/Utils/BuildLibCalls.cpp

```diff
@@ -1145,14 +1154,14 @@ bool llvm::inferNonMandatoryLibFuncAttrs(Function &F,
   case LibFunc_ldexp:
   case LibFunc_ldexpf:
   case LibFunc_ldexpl:
+    Changed |= setOnlyAccessesErrnoMemory(F);
     Changed |= setWillReturn(F);
     break;
   case LibFunc_remquo:
   case LibFunc_remquof:
   case LibFunc_remquol:
     Changed |= setDoesNotCapture(F, 2);
```

Comment:
Need to be careful with this fallthrough as well, remquo has an out param.

Reply:
Right, moving remquo together with sincos, except the latter has an extra out parameter.